examples/grpc-bridge/README.md

```diff
@@ -1,2 +1,127 @@
+* Start Both Client and Server Servers and Proxies: `
+* Use the Client CLI to make calls to the kv server.
+
+## Generate Stubs
+
+* Uses the `protos` dir and generates the stubs for both `client` and `server`
+* Inspect the file `docker-compose-protos.yaml` with the gRPC protoc commands to generate the stubs.
+
+```console
+$ docker-compose -f docker-compose-protos.yaml up --remove-orphans
+Starting grpc-bridge_stubs_python_1 ... done
+Starting grpc-bridge_stubs_go_1     ... done
+Attaching to grpc-bridge_stubs_go_1, grpc-bridge_stubs_python_1
+grpc-bridge_stubs_go_1 exited with code 0
+grpc-bridge_stubs_python_1 exited with code 0
+```
+
+* The files created were the `kv` modules for both the client and server respective dir.
+  * Note that both stubs are their respective languages.
+  * For each language, use its ways to include the stubs as an external module.
+
+```console
+$ ls -la client/kv/kv_pb2.py
+-rw-r--r--  1 mdesales  CORP\Domain Users  9527 Nov  6 21:59 client/kv/kv_pb2.py
+
+$ ls -la server/kv/kv.pb.go
+-rw-r--r--  1 mdesales  CORP\Domain Users  9994 Nov  6 21:59 server/kv/kv.pb.go
+```
+
+## Start Both Client and Server and Proxies
+
+* After the stubs are in place, start the containers described in `docker-compose.yaml`.
+
+```console
+$ docker-compose up --build
+```
+
+* Inspect the client and server envoy.yaml config on each respective dir and compare the port numbers
+
+Notice that you will be interacting with the client container, which hosts
+the client python CLI. The port numbers for the proxies and the containers are displayed
+by the `docker-compose ps`, so it's easier to compare with the envoy.yaml config for each
+of the containers how they match.
+
+Note that the client container to use is `grpc-bridge_grpc-client_1` and binds to no port
+as it will use the `python` CLI.
+
+```console
+$ docker-compose ps
+             Name                            Command               State                             Ports
+------------------------------------------------------------------------------------------------------------------------------------
+grpc-bridge_grpc-client-proxy_1   /docker-entrypoint.sh /usr ...   Up      10000/tcp, 0.0.0.0:9911->9911/tcp, 0.0.0.0:9991->9991/tcp
+grpc-bridge_grpc-client_1         /bin/sh -c tail -f /dev/null     Up
+grpc-bridge_grpc-server-proxy_1   /docker-entrypoint.sh /usr ...   Up      10000/tcp, 0.0.0.0:8811->8811/tcp, 0.0.0.0:8881->8881/tcp
+grpc-bridge_grpc-server_1         /bin/sh -c /bin/server           Up      0.0.0.0:8081->8081/tcp
+```
+
+## Use the Client CLI
+
+* Since the containers are running, you can use the client container to interact with the gRPC server through the proxies
+* The client has the methods `set key value` and `get key` to use the in-memory key-value store.
+
+```console
+$ docker exec grpc-bridge_grpc-client_1 /client/grpc-kv-client.py set foo bar
+setf foo to bar
```

Comment:
should this be `set` instead of  `setf`?

Reply:
@snowp this is the actual output of the implemented service... See same output at https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/grpc_bridge#sending-requests-to-the-key-value-store... I can fix the app itself...

Reply:
@snowp https://github.com/envoyproxy/envoy/blob/master/examples/grpc-bridge/client/client.py#L76... Should I fix this?

Reply:
Nah, that seems to fall outside the scope of this PR